### PR TITLE
feat(docs): import readme from other packages

### PR DESCRIPTION
**Pull request by sstraatemans**

### CODE REVIEW

File: packages/libs/kadena.js/README.md [hunk 1]
```diff
@@ -16,7 +16,7 @@ Kadena blockchain
 > the `local`, `development`, `testnet` or `mainnet` Kadena chainweb. It will
 > provide helper function
 
-<hr>
+---
```
Comment (webpro) [inline packages/libs/kadena.js/README.md:19]: Imho all rules can be removed :)

File: packages/apps/docs/package.json [hunk 2]
```diff
@@ -100,6 +103,7 @@
     "storybook": "^7.0.20",
     "storybook-addon-next-router": "~4.0.2",
     "storybook-dark-mode": "^3.0.0",
-    "typescript": "5.0.4"
+    "typescript": "5.0.4",
+    "dotenv": "~16.0.3"
```
Comment (webpro) [inline packages/apps/docs/package.json:107]: Please run `rush format`

File: packages/apps/docs/src/pages/_app.tsx [hunk 3]
```diff
@@ -4,6 +4,7 @@ import {
   globalCss,
 } from '@kadena/react-components';
 import { ModalProvider } from '@kadena/react-ui';
+// eslint-disable-next-line import/no-unresolved
```
Comment (webpro) [inline packages/apps/docs/src/pages/_app.tsx:7]: Shouldn't this be fixed properly?

File: packages/apps/docs/src/scripts/importReadme.mjs [hunk 4]
```diff
@@ -0,0 +1,115 @@
...
+  children: page,
+});
+
+const createDir = (dir) => {
+  const dirArr = dir.replace(DOCSROOT, '').split('/');
+  dirArr.reduce((acc, val) => {
+    const newDir = `${acc}${val}/`;
+    console.log(newDir);
+    if (!fs.existsSync(newDir)) {
+      fs.mkdirSync(newDir);
```
Comment (webpro) [inline packages/apps/docs/src/scripts/importReadme.mjs:64]: Perhaps you can use `recursive` to just create the dir (https://nodejs.org/api/fs.html#fsmkdirsyncpath-options)

File: packages/apps/docs/src/scripts/importReadme.mjs [hunk 5]
```diff
@@ -0,0 +1,115 @@
...
+
+    return acc;
+  }, []);
+};
+
+const importDocs = (filename, destination, parentTitle, RootOrder) => {
+  const doc = fs.readFileSync(`./../../${filename}`, 'utf-8');
+  const md = remark.parse(doc);
+
+  devideIntoPages(md).forEach((page, idx) => {
```
Comment (webpro) [inline packages/apps/docs/src/scripts/importReadme.mjs:88]: Typo: divide

File: packages/apps/docs/src/scripts/importReadme.mjs [hunk 6]
```diff
@@ -0,0 +1,115 @@
...
+        acc[acc.length - 1].push(val);
+      }
+    }
+
+    return acc;
+  }, []);
+};
+
+const importDocs = (filename, destination, parentTitle, RootOrder) => {
+  const doc = fs.readFileSync(`./../../${filename}`, 'utf-8');
```
Comment (webpro) [inline packages/apps/docs/src/scripts/importReadme.mjs:85]: Nitpick: wouldn't mind some more `async/await` (but def not blocking for scripts like this).

File: packages/libs/kadena.js/README.md (hunk 1 above)
Comment (sstraatemans) [reply packages/libs/kadena.js/README.md:19]: great

File: packages/apps/docs/package.json (hunk 2 above)
Comment (sstraatemans) [reply packages/apps/docs/package.json:107]: I thought that would go automatically.
not anymore?

File: packages/apps/docs/src/pages/_app.tsx (hunk 3 above)
Comment (sstraatemans) [reply packages/apps/docs/src/pages/_app.tsx:7]: Eileen is working on a fix

File: packages/apps/docs/package.json (hunk 2 above)
Comment (webpro) [reply packages/apps/docs/package.json]: `rush add` never does/did this properly

File: packages/apps/docs/src/scripts/importReadme.mjs (hunk 4 above)
Comment (sstraatemans) [reply packages/apps/docs/src/scripts/importReadme.mjs:64]: ughh...nice!

File: packages/apps/docs/src/scripts/importReadme.mjs (hunk 6 above)
Comment (alber70g) [reply packages/apps/docs/src/scripts/importReadme.mjs:77]: Counter-nitpick: async/await is slower than `sync` variants

**Review by alber70g — APPROVED** [review]

**Pull request merged**